### DISCUSSION TITLE
Warning before removing an alert

### DISCUSSION THREAD

**Discussion by Hexonium** [NightscoutFoundation/xDrip#3445] [Ideas]
When modifying an alert (in the level alerts menu), it is very easy to select "remove alert" by mistake and lose the corresponding alert. This happened to me many times while I only wanted to snooze an alert (the two buttons are next to each other). Could we have a confirmation box before removing an alert?

**Comment by Navid200** [comment]
Please explain why you access the alert edit page when you just  want to snooze the alert? 

https://navid200.github.io/xDrip/docs/Snooze.html

**Comment by Hexonium** [comment]
I use it for fine-grained snoozing (much more often than snoozing all alerts). For instance, if my BG is dropping: I can eat something, snooze the alerts above 60mg/L for 30min, but keep an alert at 40mg/L to be warned if the BG drops more than I am expecting. It's very convenient at night when I can't monitor it closely.

**Reply by Navid200** [reply]
I think it makes sense to have a confirmation pop up before deleting an alert.
Would you please click on `Create issue from discussion` in the right pane?
I'll see what I can do about it.

**Reply by Hexonium** [reply]
Thank you, done
